ops/deploy-ccs.groovy

```diff
@@ -131,10 +131,10 @@ packer build -color=false -var vault_password_file="$vaultPasswordFile" \
 			platinumAmiId: extractAmiIdFromPackerManifest(
 				readFile(file: "${workspace}/ops/ansible/playbooks-ccs/manifest_platinum.json")
 			),
-			bfdPipelineAmiId: amiIds.bfdPipelineAmiId, 
-			bfdServerAmiId: amiIds.bfdServerAmiId,
-			bfdMigratorAmiId: amiIds.bfdMigratorAmiId,
-			bfdServerLoadAmiId: amiIds.bfdServerLoadAmiId,
+			bfdPipelineAmiId: "", 
+			bfdServerAmiId: "",
+			bfdMigratorAmiId: "",
+			bfdServerLoadAmiId: "",
```

Comment:
This works just fine, but I think this is adding a bit of unnecessary technical debt in the hopes of re-using somewhat non-reusable code. The method's signature doesn't make a lot of sense outside of `ops/deploy-ccs.groovy`–consider orienting someone to this process and explaining the signature for `buildPlatinumAmi` without having to justify the relative _weirdness_ of it through examples and historical context surrounding how the process once functioned... 

Instead, this method should be removed from `ops/deploy-ccs.groovy` and the necessary logic in `jenkins/bfd-build-platinum/Jenkinsfile` should be re-defined to do whatever's necessary. That might include a new method definition or no method definition at all. Most of the necessary logic is already in the new Jenkinsfile... really, you just need to conditionally shell-out to packer and echo the resultant platinum AMI Id.

Reply:
Done